### ISSUE TITLE
Custom commands menu - tray menu icon 

### ISSUE THREAD

**Issue by Krzysztofci** [Guake/guake#1337]
Can you add custom commands menu to tray menu icon? I'm asking because the guake-indicator does not work with new versions of guake. The guake-indicator has not been updated for a long time. I am currently in the process of rewriting my commands to the custom-comands.json file. It would be nice to be able to run commands and scripts from the system tray.

Jeśli jesteście zainteresowani tym usprawnieniem głosujcie:
https://feathub.com/Guake/guake/+28
 Best regards Krzysztof

**Comment by gsemet** [comment]
Look feasible. Not sure if it make sense when guake is hidden, since it might not send the command to the right terminal.

I never used it myself, can you describe what do you use it for ?

**Comment by Krzysztofci** [comment]
I had added system update scripts to the guake-indicator (without having to update / upgrade), to change system settings, change resolutions, stop screen blanking and much more. In commands, I had a set of applications to monitor the system, or scripts to search documentation in gui zenity (in this way I am better able to read). Such a small multitool.
Generally, I have not moved all this to custom-commands.json, but below I will paste the link to the sample scripts as well as the guake-indicator.json configuration file.
https://bitbucket.org/krzysztofci/own/src/ee7a77f040bfd06422e68beb4c9b42c350cc3429?at=master
In general, this is a very useful thing and I will be very grateful if it can be implemented.

Regards

**Comment by Krzysztofci** [comment]
Thank you very much :)
Krzysztof

**Comment by Ozzyboshi** [comment]
guake indicator has been updated to work with guake, see

https://github.com/Ozzyboshi/guake-indicator/commit/6899ee202366efc91d0b5ac4f5c3fdf43120915c

The new version (1.3) accepts a new argument, -guake3 and connects to guake3 rather than guake 0.8

You want find guake indicator 1.3 on official ubuntu/debian repositories since I don't have a sponsor to upload the new version

**Comment by Krzysztofci** [comment]
The strange situation came out, so I apologize for the confusion, but I was looking at the project website and after downloading the allegedly the latest sources I saw in the changelog:
"2014-12-31 Alessio Garzi <gun101@email.it>

1.1 release with "open in selected tab" option added
"
So I doubted whether the program is still being developed. This version is also available in ubuntu 18.04.
Therefore, adding this functionality to the guake should be left to the author of the project.
Regards
Krzysztof

**Comment by Ozzyboshi** [comment]
Guake indicator is written by me, where did you find the changelog?

**Comment by Krzysztofci** [comment]
I found Changlog in the file from the page:
http://guake-indicator.ozzyboshi.com specifically is a link to these sources:
http://guake-indicator.ozzyboshi.com/sources/guake-indicator_1.1.tar.gz
There is also a changelog there. The site has been seen for a long time and has not been updated and is causing a little confusion.
Well, unless it's not the official website of the project?

**Comment by Ozzyboshi** [comment]
Yes, the official website is not updated yet with the latest release, I will do it very soon.
In addition I found a sponsor to update the debian depository.

**Comment by gsemet** [comment]
Hi Ozzyboshi! Glad to hear that :) do you think this sponsor can also takes care of guake ? It is no more updated on the Debian repositories, sadly :(

**Comment by Ozzyboshi** [comment]
When guake indicator will be ready I will ask him to review guake and upload them both if you want

**Comment by Ozzyboshi** [comment]
@Krzysztofci has solved his issues with guake-indicator, this issue can be closed unless you are really planning for a guake-indicator integration within guake.
No answer from the guake-indicator debian sponsor regarding guake3 yet.
I will try again
